apps/wallet-ui/src/lib/assets/erc721.ts

```diff
@@ -0,0 +1,86 @@
+import type { Erc721Token } from "@/types/Erc721Token";
```

Comment:
probably add `import "server-only";` since I assume this will only be called as server action / on backend?